app/src/main/java/io/left/reflect/helper/MeshHelper.java

```diff
@@ -0,0 +1,28 @@
+package io.left.reflect.helper;
+
+import io.left.rightmesh.id.MeshId;
+
+public class MeshHelper {
+
+    private static MeshHelper instance = null;
+
+    private MeshHelper() {
+    }
+
+    public static MeshHelper getInstance(){
+        if(instance == null)
+            instance = new MeshHelper();
+
+        return instance;
+    }
+
+    /**
+     * Truncates MeshIds to 8 characters long.
+     *
+     * @param id to get string of
+     * @return truncated string
+     */
+    public String shortenMeshId(MeshId id) {
```

Comment:
I think the singleton can help for lazy initialization but I will make it thread-safe in the next version. 
May I know the purpose of this project? I think it could be better to apply Dagger 2 if this project is our demo app and we have plans to improve it. However, if this is a way to introduce for users how to use Rightmesh library, I'll keep it as simple as possible.